templates/committees-single.html

```diff
@@ -10,7 +10,11 @@
       <div class="container">
         <div class="entity__header--top row">
             <h1 class="entity__name">{{ name }}</h1>
-            <span class="entity__type"><img class="category-icon" width="20" height="20" src="/static/img/icon-committee--white.svg" alt="Icon representing committees">Committee - {{ designation }}</span>
+            <span class="entity__type">
+              <img class="category-icon"
+                   width="20" height="20"
+                   src="/static/img/icon-committee--white.svg"
+                   alt="Icon representing committees">Committee - {{ designation }}</span>
```

Comment:
We want that `.entity__type` to actually concatenate the three elements, so it would look like:

`Committee -- Principal Campaign Committee -- Presidential`

or

`Committee -- PAC -- Organized Labor`